lx_administration/models/vault/ansible_cfg.py

```
@@ -87,11 +89,11 @@ def drop_missing_vid(self):
         self.vault_identity_list = self.vid_list2str(self.vid_dict2list(vid_dict))
 
         if file_not_found:
-            ic(
+            warnings.warn(
```

Ruff (0.13.3)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)